config_operator/config_operator/k8s/K8sConfigMap.py

```diff
@@ -0,0 +1,97 @@
+import os
+import logging
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+
+from config_operator.config_source.exceptions import UnreadableFileException
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class K8sConfigMap:
+    def __init__(self, configmap_name, namespace, git_remote_config):
```

Comment:
Can `namespace` be detected from the Kubernetes client? The user shouldn't have to explicitly pass in the namespace since the configmap should just get published in the same namespace that this pod is running in. (Or the `--namespace` command line argument should be made optional in case the user wants to override it.)

Reply:
Yes, you are right this was not good to request the namespace here. This is solved by the k8s operator which reads the namespace automatically in its code.